src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Address.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Address]
+(
+	AddressId INT IDENTITY(1,1) NOT NULL PRIMARY KEY, 

```

Comment:
Can you add a space after the comma in the `IDENTITY` function so it becomes `IDENTITY(1, 1)`? (Just slightly more readable.)

Reply:
👍